Firestore/core/test/firebase/firestore/remote/serializer_test.cc

```diff
@@ -14,6 +14,26 @@
  * limitations under the License.
  */
 
+/* NB: proto bytes were created via:
+     echo 'TEXT_FORMAT_PROTO' \
+       | ./build/external/protobuf/src/protobuf-build/src/protoc \
+           -I./Firestore/Protos/protos \
+           -I./build/external/protobuf/src/protobuf/src/ \
```

Comment:
Nit: you're mixing trailing slashes and non-trailing slashes in the same command.

Reply:
Done